packages/job-worker/src/playout/playout.ts

```diff
@@ -783,7 +783,7 @@ async function _onPartPlaybackStarted(
 		throw new Error(`PartInstance "${data.partInstanceId}" in RundownPlayst "${data.playlistId}" not found!`)
 
 	// make sure we don't run multiple times, even if TSR calls us multiple times
-	const isPlaying = playingPartInstance.timings?.startedPlayback && !playingPartInstance.timings?.stoppedPlayback
+	const isPlaying = !!playingPartInstance.timings?.startedPlayback
```

Comment:
this should probably be renamed to `hasStartedPlaying`, as it could have started and already stopped